lib/styles/themes/base/def/index.js

```diff
@@ -0,0 +1,24 @@
+// Import the Design System Tokens for current theme: Base
+import '../tokens/index.css';
+
+import animations from './animations';
+import behaviour from './behaviour';
+import border from './border';
+import colors from './colors';
+import dimensions from './dimensions';
+import typography from './typography';
+import zIndex from './zIndex';
+import shadow from './shadow';
+import spacing from './spacing';
+
+export default {
```

Comment:
Why not expose them separately. 

The way you will be eventually consuming this will be 

theme.spacing.something else which may not be ideal due to couple of reasons. 
These are JS imports, if you don't want animations and behaviour, you anyways get them as a packaged deal.

We can talk about it

Reply:
Not much to gain, all of this get consumed in CSS. Also ThemeProvider accepts a theme object, we will have to combine it there and maintain the configuration of passing whole theme entities in rest of the uses.